### PR TITLE
Clarify agent events

### PR DESCRIPTION
**Pull request by jradtilbrook**
Adds more details about what each event means and how it occurs.

Before:
Screen Shot 2021-11-25 at 10 19 48

After:
Screen Shot 2021-11-25 at 10 19 24

### CODE REVIEW

**Comment by karensawrey** [comment]
The original info repeats in:
https://buildkite.com/docs/apis/webhooks/agent-events#events and
https://buildkite.com/docs/apis/webhooks#events

Should we turn the updated info into reusable snippets, @plaindocs?

File: pages/apis/webhooks/agent_events.md.erb [hunk 1]
```diff
@@ -8,23 +8,23 @@
 <tbody>
   <tr>
     <th><code>agent.connected</code></th>
-    <td>An agent has connected</td>
+    <td>An agent has connected to the API</td>
   </tr>
   <tr>
     <th><code>agent.lost</code></th>
-    <td>An agent has been marked as lost  </td>
+    <td>An agent has been marked as lost. This happens when Buildkite stops receiving heartbeats from the agent.</td>
```
Comment (plaindocs) [inline pages/apis/webhooks/agent_events.md.erb:15]: Worth checking if we talk about heartbeats elsewhere. In this case we don't use the term else where. Is this the term we should be using. Would "pings" work?

File: pages/apis/webhooks/agent_events.md.erb [hunk 2]
```diff
@@ -8,23 +8,23 @@
...
   </tr>
   <tr>
     <th><code>agent.disconnected</code></th>
-    <td>An agent has disconnected</td>
+    <td>An agent has disconnected. This happens when the agent disconnects from the API.</td>
```
Comment (plaindocs) [inline pages/apis/webhooks/agent_events.md.erb:19]: and is different from having no internet connection, say?

**Review by plaindocs — COMMENTED** [review]
These are EXCELLENT additions!

@karensawrey yeah, definitely. Happy to pick that up?

**Comment by karensawrey** [comment]
I'll be happy to turn these repeated bits into reusable snippets, @plaindocs!
@jradtilbrook let's clarify the 2 remaining unclear parts & I'll make sure it's broadcasted to all the places it's needed, from a single source.

File: pages/apis/webhooks/agent_events.md.erb (hunk 1 above)
Comment (jradtilbrook) [reply pages/apis/webhooks/agent_events.md.erb:15]: Oh yeah "pings" is probably better
```suggestion
    <td>An agent has been marked as lost. This happens when Buildkite stops receiving pings from the agent.</td>
```

File: pages/apis/webhooks/agent_events.md.erb (hunk 2 above)
Comment (jradtilbrook) [reply pages/apis/webhooks/agent_events.md.erb:19]: Or perhaps below is clearer, to distinguish between stopping (API tells it what to do) and disconnecting (agent decides what to do)

```suggestion
    <td>An agent has disconnected. This happens when the agent shuts down and disconnects from the API.</td>
```

**Comment by jradtilbrook** [comment]
@karensawrey @plaindocs sounds good. I've added some suggestions to the comments

**Comment by karensawrey** [comment]
Great work, folks! I'll propagate the changes via snippets ASAP. Merging now.

**Pull request merged**